react/components.md

```diff
@@ -0,0 +1,96 @@
+### Introduction
+
+Introduction to React components, what they do, and how to write them. In this lesson we'll be going over the basics of React components - what they do, and how to write them. Make sure to use the project you set up in the [Setting Up A React Environment lesson](https://github.com/TheOdinProject/top-meta/issues/221), but try not to copy and paste any code while you're coding along. 
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+*   What are components?
+*   How are components created?
+*   Where do components live?
+
+### What are components
+
+The beauty of React is that it allows you to break a UI (User Interface) down into independent reusable chunks, which we will refer to as components. If it helps, think of these reusable chunks as JavaScript functions which can take some kind of input and return a React element. Before we go any further, let's look at a React element. 
+
+~~~jsx
+const element = <h1>The Odin Project rocks!</h1>;
+~~~
+
+Beautiful! But what exactly is happening here? We're using JSX to produce an element, and assigning it to the constant variable `element`. We'll go into JSX more in the next lesson, but for now it's enough to know that JSX is essentially an HTML-*like* syntax which is converted into JavaScript. 
+
+If you haven't already, go back to your previous React project and display "Hello, World!" by creating a React element, and passing it as an argument into your project's `root.render()` function.
```

Comment:
I'm unsure what the React element example is achieving? And the suggestion to display a "Hello, World!" element with `root.render()` sounds confusing.